src/tool/subcommands/api_cmd.rs

```diff
@@ -465,6 +465,17 @@ fn snapshot_tests(store: &ManyCar, n_tipsets: usize) -> anyhow::Result<Vec<RpcTe
     tests.extend(state_tests(&shared_tipset));
     tests.extend(eth_tests_with_tipset(&shared_tipset));
 
+    // Those proven hard to verify as most of the time both lotus and forest return 'null'. So it's
+    // hardcoded.
```

Comment:
this doesn't quite make sense to me - consider rewording?

Reply:
This note implies understanding of snapshot tests, which takes `n` last tipsets and tries to verify the results for both Lotus and Forest using the addresses found. Since last `n` tipsets data doesn't return anything but `null` for both - the addresses are hardcoded.